src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/auth/register/AttachedRegisterPage.java

```diff
@@ -42,7 +42,12 @@ public AttachedRegisterPage() {
 
   @Override
   public AttachedRegisterPage open() {
-    driver.get(urlBuilder.getUrl() + URLsContent.USER_SIGNUP);
+    try {
+      driver.get(urlBuilder.getWikiGlobalURL() + URLsContent.USER_SIGNUP + "?redirect=" + URLEncoder
+          .encode(urlBuilder.getUrl(), "UTF-8"));
+    } catch (UnsupportedEncodingException e) {
+      e.printStackTrace();
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Use a logger to log this exception. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AS1148)